android/titanium/src/java/org/appcelerator/titanium/TiFileProxy.java

```diff
@@ -139,6 +138,13 @@ public boolean getWritable()
 	}
 
 	@Kroll.method
+	public boolean append(Object data)
```

Comment:
There was only 1 write method. The original write method had only one argument (Object[] args). I changed it so that it could take in append argument which is necessary. It keeps the original write exposed, the new write method not exposed and append is exposed too. But I'll be sure to change this one line.